### PR TITLE
fixed typos and formatting in the docs

### PR DESCRIPTION
**Pull request by antoniouaa**
This addresses issue #61.

Some formatting and rewording in the docs

I noticed that in [docs/database_api.md](docs/database_api.md) you have this:

### Classifier prediction files metadata

```json
{
    "F1": "0.7030995388400528",
    "accuracy": "0.7034883720930233",
    "classificator": "nb",
    "filename": "titanic_testing_new_prediction_nb",
    "fit_time": 41.870062828063965
}
```

* `F1` - F1 Score from model accuracy
* `accuracy` - Accuracy rate from model prediction
* `classificator` - Initials from used classificator
* `error` - Error rate from the model prediction
* `fit_time` - Time taken for the model to be fit during training

The fields breakdown has the `error` field that doens't exist in the JSON response.
I'm not sure what to do with that.

### CODE REVIEW

**Comment by riibeirogabriel** [comment]
@antoniouaa It is a doc bug, "error" was a metric which JSON return in old version of Learning Orchestra, In current release the "error" field are not returned, please replace the "error" field by "filename".

File: docs/index.md [hunk 1]
```diff
@@ -6,54 +6,38 @@
...
-learningOrchestra's [collection of microservices](https://riibeirogabriel.github.io/learningOrchestra/usage).
+It is also possible to perform several preprocessing and analytical tasks using learningOrchestra's [collection of microservices](https://riibeirogabriel.github.io/learningOrchestra/usage).
 
-With learningOrchestra, you can build prediction models with different 
-classificators simultaneously using stored and preprocessed datasets with the
-*Model Builder* microservice. This microservice uses a [Spark](https://spark.apache.org/)
-cluster to make prediction models using distributed processing. You can compare the different 
-classification results over time to fit and increase prediction accuracy.
+With learningOrchestra, you can build prediction models with different classifiers simultaneously using stored and preprocessed datasets with the *Model Builder* microservice. This microservice uses a [Spark](https://spark.apache.org/) cluster to make rediction models using distributed processing. You can compare the different classification results over time to fit and increase prediction accuracy.
```
Comment (riibeirogabriel) [inline docs/index.md:27]: Please replace misspelled "rediction" to "prediction"

File: readme.md [hunk 2]
```diff
@@ -6,9 +6,7 @@
 
 ## Objective
 
-The goal of this work is to develop a tool, named *learningOrchestra*, to
-facilitate and streamline the data science iterative process of:
-
+The goal of this work is to develop a tool, named *learningOrchestra*, to facilitate and streamline the data science iterative rocess of:
```
Comment (riibeirogabriel) [inline readme.md:9]: please replace "rocess" to "process"

**Review by riibeirogabriel — COMMENTED** [review]
Please take a look from my annotations in your PR.

**Comment by antoniouaa** [comment]
Oopsie.

Typos fixed.

File: docs/database_api.md [hunk 3]
```diff
@@ -88,44 +79,40 @@ this filename.
...
 
-* F1 - F1 Score from model accuracy
-* accuracy - accuracy rate from model prediction
-* classificator - initials from used classificator
-* error - error rate from model prediction
-* fit_time - time from model fit using training dataset
+* `F1` - F1 Score from model accuracy
+* `accuracy` - Accuracy rate from model prediction
+* `classificator` - Initials from used classificator
+* `error` - Error rate from the model prediction
```
Comment (riibeirogabriel) [inline docs/database_api.md:85]: Please replace `error` field by `filename` field

Comment (riibeirogabriel) [reply docs/database_api.md:85]: You can copy the `filename` definition from any other JSON microservice.

Comment (antoniouaa) [reply docs/database_api.md:85]: Alright, I was wondering why I couldn't see error in the JSON

**Review by riibeirogabriel — APPROVED** [review]
Seems good, thanks!

**Pull request merged**